x-pack/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -71,7 +72,9 @@ const HostDetailsLinkComponent: React.FC<{ children?: React.ReactNode; hostName:
     [hostName, navigateToApp, search]
   );
 
-  return (
+  return isButton ? (
+    <EuiButton onClick={goToHostDetails}>{children ? children : hostName}</EuiButton>
```

Comment:
I could style the link as a button, but figured this was more straightforward. Is there a reason why both the `href` and `goToHostDetails` are needed? It looks like `goToHostDetails` also changes the url params?

Reply:
I think we wanted that if people want to right click on the link and open it in a new tab, you will go directly to the host details

Reply:
but now I can see we do not want that since we have the flyout and then you can go to the details page by clicking on it or right click on it ;)

Reply:
Good to know, thanks! Updated to use the `LinkButton`